packages/ui/resolution/ResolutionDecorator.js

```diff
@@ -97,7 +97,22 @@ const ResolutionDecorator = hoc(defaultConfig, (config, Wrapped) => {
 		static displayName = 'ResolutionDecorator';
 
 		static propTypes = /** @lends ui/resolution.ResolutionDecorator.prototype */ {
-			className: PropTypes.string
+			className: PropTypes.string,
+
+			/**
+			 * Font Scale for large text mode.
+	 		 * Use this value to set the scale of the font.
+	 		 * This is the value that will be multiplied by pxPerRem, which is determined by resolution.
```

Comment:
I used it because the currently used term is large text mode. It may be confused with large text in ThemeDecorator, but since there is currently no fixed name, it would be a good idea to change it after the name is fixed. This is a draft and will not be merged.

Reply:
I agree. I think we need to set a proper name for this mode when merging it. I will approve it since this is a draft PR.